amplipi/utils.py

```diff
@@ -377,19 +378,19 @@ def get_identity() -> dict:
       potential_identity = json.load(identity_file)
       identity.update(potential_identity)
   except FileNotFoundError as e:
-    print(f'Error loading identity file: {e}')
-    print('Creating an identity file from defaults.')
+    logging.exception(f'Error loading identity file: {e}')
+    logging.info('Creating an identity file from defaults.')
```

Comment:
good example of where using two different log levels is helpful; the second message informs a user what is going on, with reasonable context.